airqo-platform/AirQo-api
# Ft device status summary

What does this PR do?
This PR creates a new endpoint for retrieving device summary (no. of devices, no. of devices on solar, no. of devices on electricity)
What JIRA task is related to this PR?
The link to the JIRA board task is [here](https://airqoteam.atlassian.net/browse/NET-10).
How do I test out this PR?
`Git clone` this repo `cd AirQo-api/src/device-monitoring`. Then setup python virtual environment using the following commands: `python3 -m venv env`, followed by `source env/bin/activate`.  Next, create a `.env` file and paste in the content provided by the developer. Now, run `pip install -r requirements/dev.txt` to get all the required packages. Afterwards, run `flask run` to test out the endpoints using any REST client tool available to you. 
Which endpoints should be/are ready for testing?
partial URLs:
`/api/v1/device/monitor/status`
